README.md

```diff
@@ -1,3 +1,5 @@
+<h1><img src="images/grubs_logo.png" alt="Grubs logo" height="200"/></h1>
```

Comment:
I'd rather use something that looks like it's designed to be used a header. I'll send it to you over Discord when I have it.